File: python/mod_8/task_5.py
def main():
    print("=====Кенийский бегун=====")
    
    while True:
        temp = 0
        km = 0
        print("Ввод: ")
        distanse = float(input("Привет, Сэм! Сколько километров ты сегодня пробежал? "))

        while km < distanse:
            sec = float(input(f"Какой у тебя был темп на километре {km+1}? "))
            temp += sec
            km += 1
        
        temp = round(temp/distanse)
        sec = temp%60
        temp /= 60

        print("Вывод: ")
        print(f"Твой средний темп за тренировку: {int(temp)} минут и {sec} секунд.")

if __name__ == "__main__":
    main()

File: final/mod_8/task_5.cpp
#include <iostream>
#include <cmath>

int main(){
    using namespace std;
    cout << "=====Кенийский бегун=====\n";

    int distance;
    float sec, temp;

    while (true){
        cout << "Ввод:\n";
        cout << "Привет, Сэм! Сколько километров ты сегодня пробежал? "; cin >> distance;
        for (int i=0; i < distance; i++){
            cout << "Какой у тебя был темп на километре " << i+1 << "? "; cin >> sec;
            temp += sec;
        }
        temp = round(temp/distance);
        sec = (int)temp % 60;
        temp /= 60;

        cout << "Вывод:\n";
        cout << "Твой средний темп за тренировку: " << (int)temp << " минуты и " << sec << " секунд.\n";
        cout << "==========\n";




    }


}